packages/data/package.json

```diff
@@ -23,6 +23,7 @@
 	"dependencies": {
 		"@woocommerce/date": "2.1.0",
 		"@woocommerce/navigation": "5.2.0",
+		"@wordpress/i18n": "3.17.0",
```

Comment:
The `woocommerce/date` package relies on an older version of `i18n`, which has a wrong TS type, the newer version has the correct type.